gazebo-odometry/src/odometry_node.cpp

```diff
@@ -6,19 +6,21 @@
 
 int main(int argc, char** argv) {
     ros::init(argc, argv, "odometry");
-    ros::NodeHandle nh;
-    const int timeout nh.param("timeout", 250);
+    ros::NodeHandle nh("~");
```

Comment:
i think you may have mixed up which is which here, the private handle has the "~" namespace